Tests/UnitTests/Utilities/Session/PersistentSessionManagerTests.swift

```diff
@@ -289,6 +291,7 @@ extension PersistentSessionManagerTests {
         // WHEN I clear all session data
         try sut.clearAllSessionData()
         // THEN my session data is deleted
+        XCTAssertFalse(mockWalletAvilibilityService.hasAccessedBefore)
```

Comment:
Please can we set this to true in the `GIVEN` section above, so we can be sure that it is reset?
Otherwise I think we're just checking that `false` stays `false`? 🙂

Reply:
I had to remove this fully, because the way it was set-up, the delete function has been overridden for `SessionBoundData` to return true when called if successful - I think this should be covering the wallet data as well as any other data passed in as `SessionBoundData`

Reply:
I'm not sure what you mean, I think we just need to add:
`mockWalletAvailabilityService.hasAccessedBefore = true`, before clearing the session data otherwise that's already false

Reply:
Talked with Oli - will be picked up at a later time to add a test for this.